release-notes/release-notes53.md

```diff
@@ -63,42 +63,15 @@ For a list of valid menu items, see: [Editor control identifiers - Menu controls
 
 ### Enhanced Media Embed
 
```

Comment:
Where did all of these notes go? Somewhere else?

Reply:
I guess this was very repetitive so it got refactored into the shorter section below? The headings appear to still exist though...

Reply:
Sorry, I was consolidating the note for these, and forgot to add something to these sections. Fixed in #1520